crates/bevy_core_pipeline/src/lib.rs

```diff
@@ -81,9 +81,6 @@ impl Plugin for CorePipelinePlugin {
                 OrderIndependentTransparencyPlugin,
                 MipGenerationPlugin,
             ));
-    }
-
-    fn finish(&self, app: &mut App) {
```

Comment:
does this work? thats surprising

Reply:
Yup, we just initialize the full screen shader resource which just needs the asset server.

Reply:
I just realized. There's no FromWorld so its just standard init